docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -0,0 +1,44 @@
+---
+id: nexus
+title: Temporal Nexus - Temporal feature
+description: Explore how Temporal Nexus enables modular architecture and secure multi-team collaboration by abstracting Temporal primitives into clean Nexus Service APIs across namespace boundaries.
+sidebar_label: Temporal Nexus
+slug: /temporal-nexus
+tags:
+  - temporal-nexus
+  - temporal-cloud
+
+keywords:
+  - temporal multi-team collaboration
+  - modular temporal architecture
+  - namespace boundary workflows
+  - temporal nexus api registry
+  - temporal nexus services
+  - secure temporal workflows
+  - multi-team integration temporal
+  - temporal cloud nexus
+---
+
+:::tip Support, stability, and dependency info
+
+Temporal Nexus is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for both OSS and Temporal Cloud.
+
+:::
+
+
+Temporal Nexus is an integrated feature of the Temporal platform designed to connect durable executions across team and Namespace boundaries.
```

Comment:
```suggestion
Temporal Nexus is a new feature of the Temporal platform designed to connect durable executions across team, namespace, region, and cloud boundaries. It promotes a more modular architecture for sharing a subset of your team’s capabilities via well-defined service API contracts for other teams to use, that abstract underlying Temporal primitives, like Workflows, or execute arbitrary code.```

Reply:
taking away the word `new` since it will end up dating the docs.